### PR TITLE
Support Symfony 3.4

### LINKED ISSUE #4: No support for Symfony 3.4
Symfony 3.4 is still supported until Nov. 2021. https://symfony.com/roadmap/3.4

We would like to use this nice library but #2 forces the user of this library also to use symfony 4.1 which currently is not in use here as we rely on LTS versions.

Would it be okay (in the sense of would you accept a pull request) to have the requirement for `symfony/process` to something like `^3.4|^4.1`?

Thanks.

### PR DESCRIPTION
**Pull request by BreiteSeite**
Fixes #4

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,20 +3,33 @@ language: "php"
...
+          env: PREFER_LOWEST=1
+        - php: '7.3'
+          env: PREFER_LOWEST=1
 
 before_install:
     - "composer self-update"
 
 install:
-    - "composer install"
+    - if [[ $PREFER_LOWEST != 1 ]]; then composer update --prefer-dist --prefer-stable --no-interaction; fi
```
Comment (schroedan) [inline .travis.yml:21]: Is it really necessary to run `composer update` when `$PREFER_LOWEST != 1`? Please use `composer install --prefer-dist --no-interaction` instead.

Comment (BreiteSeite) [reply .travis.yml:21]: I tried that, see commit 977aea8 (using composer install and failing) and 7aa518b (using composer update and passing).

I'm not sure if it's valuable to commit the lock anyway for a library (see https://getcomposer.org/doc/02-libraries.md#lock-file). In the end i think it makes sense to test against the lowest and highest possible versions and not against the version someone once updated or installed.

Comment (schroedan) [reply .travis.yml:21]: Okay. So then please remove the lock file and put it under `.gitignore`.

Comment (BreiteSeite) [reply .travis.yml:21]: Thank you! I didn't do this to keep the scope as small as possible but i like it much better without the `composer.lock` file.

**Review by schroedan — APPROVED** [review]

**Pull request merged**